### ISSUE TITLE
Make this server side compatible

### ISSUE THREAD

**Issue by bkniffler** [oneuijs/oui-dom-utils#1]
This line makes oui-dom-utils throw an error on server side (and all libraries that rely on it like https://github.com/recharts/recharts/)

```
const requestAnimationFrame = window.requestAnimationFrame ||
window.webkitRequestAnimationFrame ||
window.mozRequestAnimationFrame;
```

Could you wrap it somewhere? It is only used in scrollTo anyways.

**Comment by camsong** [comment]
Should be fixed with the release of [v0.3.6](https://github.com/oneuijs/oui-dom-utils/releases/tag/v0.3.6)

**Comment by bkniffler** [comment]
Excellent, thanks!
